nextjs/src/app/(auth)/login/login-form.tsx

```diff
@@ -92,8 +98,10 @@ export default function LoginForm() {
           )}
         />
         <FormMessage />
-        <Button variant='reverse' type="submit" disabled={loading}>Login</Button>
-      </form>
+        <Button variant='reverse' type="submit" disabled={loading} className="flex items-center justify-center">
+          {(loading || loginAttemptStarted) ? (<>Loading...</>) : 'Login'}
```

Comment:
comment: I don't think this will work quite right. If the login attempt fails, `loginAttemptStarted` will still be `true`, so the `Loading` indicator will still appear.

suggestion: remove `loginAttemptStarted`, and instead use `loading || token` here. This will show `Loading` if the request is in progress, or if it was successful.